code/datums/diseases/advance/symptoms/wizarditis.dm

```diff
@@ -0,0 +1,116 @@
+/datum/symptom/wizarditis
+	name = "Wizarditis"
+	desc = "Causes the host's brain cells to naturally die off, causing severe brain damage."
+	stealth = 1
+	resistance = -2
+	stage_speed = -3
+	transmittable = -1
+	level = 0
+	severity = 3
+	symptom_delay_min = 15
+	symptom_delay_max = 30
+	var/teleport = FALSE
+	var/robes = FALSE
+	threshold_desc = "<b>Transmission 14:</b> The host teleports occasionally.<br>\
+					  <b>Speed 7:</b> The host grows a set of wizard robes."
+
+/datum/symptom/wizarditis/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["transmission"] >= 14)
+		teleport = TRUE
+	if(A.properties["speed"] >= 7)
+		robes = TRUE
+
+/datum/symptom/wizarditis/Activate(datum/disease/advance/A)
```

Comment:
Going to hit this entire proc with a 🤔

Reply:
Need to know what specifically to fix to fix it

Reply:
Eh, don't worry about it. Not a huge deal.